mailserver/mailserver.go

```diff
@@ -450,16 +388,16 @@ func (s *WMailServer) SyncMail(peer *whisper.Peer, request whisper.SyncMailReque
 // exceedsPeerRequests in case limit its been setup on the current server and limit
 // allows the query, it will store/update new query time for the current peer.
 func (s *WMailServer) exceedsPeerRequests(peer []byte) bool {
-	s.muLimiter.RLock()
-	defer s.muLimiter.RUnlock()
+	s.muRateLimiter.RLock()
+	defer s.muRateLimiter.RUnlock()
 
-	if s.limiter != nil {
+	if s.rateLimiter != nil {
```

Comment:
I would invert this `if` to avoid unnecessary nesting

Reply:
I can change it :)